src/suites/cts/validation/vertex_input.spec.ts

```diff
@@ -564,9 +652,15 @@ g.test('check attribute offset out of bounds', async t => {
     const descriptor = t.getDescriptor(vertexInput, VERTEX_SHADER_CODE_WITH_NO_INPUT);
     t.device.createRenderPipeline(descriptor);
   }
+  {
+    // Control case, setting attribute offset to 4
+    vertexInput.vertexBuffers![0].attributeSet![0].offset = 4;
```

Comment:
Another good control case would be the largest valid value: offset=MAX_VERTEX_BUFFER_END-4. Or if the format were float2, MAX_VERTEX_BUFFER_END-8.

Reply:
This is actually the first control case, I've updated the "not that clear" comment.

Reply:
Oh yes, sorry I missed that.